lib/runner.go

```diff
@@ -330,23 +331,33 @@ func (r *Runner) printVerboseSingleTestResult(result TestResult) {
 
 func (r *Runner) collectFailedTestResults() {
 	for _, result := range r.testResults {
-		if !result.Success {
+		if !result.Success && !result.Skipped {
```

Comment:
Note. We should IMHO also collect the skipped tests and then report them at the end, like for failed tests.

Reply:
I initially thought about it, but then I came to the conclusion that if the test skips, we know exactly why, and it is not a failure that should be investigated. If the person running the test wants to know the output of the skipped test, pass `--verbose`.

Reply:
This is not about getting the entire log from skipped tests.
I simply want to know which tests skipped, their names instead of just that X tests skipped.
More situational awareness about things without going overboard on the output.
I.e. maybe I have forgotten to start credhub, and then seeing that this test skipped alerts me to it.

Reply:
Oh, this is implemented already.
https://github.com/SUSE/testbrain/pull/25/files#diff-ce53f23fc2093d441d80826073c7d2eeR319

Reply:
No, I do not believe this is it.
This is the main list of tests printed while the brain is running.

When tests fail the runner prints an additional short list of these failed tests at the end.
I would like to have this short list at the end for skipped tests as well.

Reply:
Gotcha.

Reply:
Here is another reason for this summary list.
I ran the brain tests with defaults => Apparently that is verbose.

The main `SKIPPED` lines are now deeply in the log file, pretty much invisible.

Reply:
On SCF it will change to non-verbose as default here https://github.com/SUSE/scf/pull/2172.

Reply:
I'll implement the summary anyways! I can clearly see the reason for it.